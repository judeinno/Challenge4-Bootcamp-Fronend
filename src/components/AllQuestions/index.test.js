import React from 'react';
import { shallow } from 'enzyme';
import QuestionsDisplay from './index';

it('testing QuestionsDisplay', () => {
  const questionData = { question: { Question: 'How to train a Dragon?' } };
  const QuestionsDisplayComponent = shallow(
    <QuestionsDisplay question={questionData} />,
  );
  expect(QuestionsDisplayComponent).toMatchSnapshot();
});
