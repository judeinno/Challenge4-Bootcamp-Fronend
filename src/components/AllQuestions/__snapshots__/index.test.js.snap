// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing QuestionsDisplay 1`] = `
<div
  className="container question-card"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <p
        className="card-text"
      />
      <a
        className="btn btn-primary"
        href="#"
      >
        More
      </a>
    </div>
  </div>
</div>
`;
